sneaky-foxes/obsidian-safe-filename-linter
# Add equal sign to Bad Character Options

**Description**

Adds a new section and the `=` character

<img src="https://github.com/sneaky-foxes/obsidian-safe-filename-linter/assets/6129479/a649cfaa-2c4f-400d-b68d-e03516b15f2e" width="400">

**Tested on**

Ubuntu 22.04

**Testing Steps**

- Create a filename with an equal 
- Open the options and enable replacing equal
- Run `Safe Filename Linter: Lint the Current File`
- Observe rename

**Related Issue**
#14 

**Note**

I couldn't find any documentation of OneDrive disallowing equal signs, but I use neither system and its an easy add, so I'll take their word!
